src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `box-sizing` property should be defined using the `*` selector, and applied to all elements, not just the `body`. This ensures that all elements on the page are sized in the same way.